posixlib/src/main/scala/scala/scalanative/posix/time.scala

```diff
@@ -165,6 +159,7 @@ trait time extends libc.time {
 object timeOps {
   import time.{time_t, timespec, tm}
 
+  // duplicate of libc.time.timespecOps
```

Comment:
Not the biggest wart and there is an appropriate
warning comment.

Is there something about implicits which mean that
the posix timespecOps can not extend the proposed
libc.time.timespecOps so that Single Point of Truth
is maintained?